integration/tests/cook/test_basic.py

```diff
@@ -67,6 +67,50 @@ def test_basic_submit(self):
         else:
             self.logger.info(f'Exit code not checked because cook executor was not used for {instance}')
 
+    def test_submit_requesting_gpus(self):
+        settings_dict = util.settings(self.cook_url)
+        gpu_enabled = settings_dict['mesos-gpu-enabled']
+        if not gpu_enabled:
+            self.skipTest("Cluster is not gpu-enabled")
+        else:
+            valid_gpu_models_config_map = settings_dict.get("pools", {}).get("valid-gpu-models", [])
+            if not valid_gpu_models_config_map:
+                self.skipTest("No pools have valid-gpu-models")
+            else:
+                active_pools, _ = util.active_pools(self.cook_url)
+                if len(active_pools) == 0:
+                    self.logger.info('There are no pools to submit jobs to')
+                    self.skipTest("There are no active pools that support GPUs")
+                for pool in active_pools:
+                    pool_name = pool['name']
+                    matching_gpu_models = [ii["valid-models"] for ii in valid_gpu_models_config_map if
+                                           re.match(ii["pool-regex"], pool_name)]
+                    if len(matching_gpu_models) == 0 or len(matching_gpu_models[0]) == 0:
+                        self.skipTest(f"Pool {pool} does not support GPUs")
+                    else:
+                        job_executor_type = util.get_job_executor_type()
```

Comment:
Is this actually needed? I think the test will work without specifying the executor.

Reply:
deleted